pages/common/git-flow.md

```diff
@@ -0,0 +1,24 @@
+# git flow
+
+> A collection of Git extensions to provide high-level repository operations for Vincent Driessen's branching model.
```

Comment:
Is it really important to add "Vincent Driessen's branching model" ? I think the meaning still stands without it.

Reply:
true